docs/dev-tools/chopsticks/overview.md

```diff
@@ -0,0 +1,412 @@
+|     `build-block-mode`     |                       How blocks should be built in the fork: batch, manual, instant                        |
+|      `import-storage`      |              A pre-defined JSON/YAML storage file path to override in the parachain's storage               |
+| `allow-unresolved-imports` |              Whether to allow WASM unresolved imports when using a WASM to build the parachain              |
+|           `html`           |                           Include to generate storage diff preview between blocks                           |
+|   `mock-signature-host`    | Mock signature host so that any signature starts with `0xdeadbeef` and filled by `0xcd` is considered valid |
+
+### Using a Configuration File
+
+The Chopsticks source repository includes a collection of [YAML](https://yaml.org/){target=_blank} files that can be used to set up various Substrate chains locally. You can download these configuration files from the [repository's `configs` folder](https://github.com/AcalaNetwork/chopsticks/tree/master/configs){target=_blank}.
+
+An example of a configuration file for Polkadot is as follows:
+
+```yaml
+endpoint:
+  - wss://rpc.ibp.network/polkadot
+  - wss://polkadot-rpc.dwellir.com
+mock-signature-host: true
+block: ${env.POLKADOT_BLOCK_NUMBER}
+db: ./db.sqlite
+runtime-log-level: 5
+
+import-storage:
+  System:
+    Account:
+      -
+        -
+          - 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+        - providers: 1
+          data:
+            free: '10000000000000000000'
+  ParasDisputes:
+    $removePrefix: ['disputes'] # those can makes block building super slow
+```
+
+To run Chopsticks using a configuration file, utilize the `--config` flag. You can use a raw GitHub URL, a path to a local file, or simply the chain's name. For example, the following commands all use Polkadot's configuration in the same way:
+
+=== "GitHub URL"
+
+    ```bash
+    npx @acala-network/chopsticks \
+    --config=https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/polkadot.yml
+    ```
+
+=== "Local File Path"
+
+    ```bash
+    npx @acala-network/chopsticks --config=configs/polkadot.yml
+    ```
+
+=== "Chain Name"
+
+    ```bash
+    npx @acala-network/chopsticks --config=polkadot
+    ```
+
+!!! note
+    If using a file path, make sure you've downloaded the [Polkadot configuration file](https://github.com/AcalaNetwork/chopsticks/blob/master/configs/polkadot.yml){target=_blank}, or have created your own.
+
+### Using Command Line Interface (CLI)
+
+Alternatively, all settings (except for genesis and timestamp) can be configured via command-line flags, providing a comprehensive method to set up the environment. For example, the following command forks Polkadot at block 100.
+
+```bash
+npx @acala-network/chopsticks \
+--endpoint wss://polkadot-rpc.dwellir.com \
+--block 100
+```
+
+## Interacting with a Fork
```

Comment:
Should we add that if you run multiple forks they are automatically running in 8000, 8001, etc? And that you can set the port you want to run the fork on with `--port`?

Reply:
It is already mentioned in the Configuration section. Maybe we could add the multiple forks scenario